components/x-gift-article/src/lib/highlightsApi.js

```diff
@@ -54,4 +56,20 @@ export default class HighlightsApiClient {
 			return { annotationsCopyResult: undefined }
 		}
 	}
+
+	async shareHighlights(articleId) {
+		try {
+			const response = await this.fetchJson('/create-token', {
+				method: 'POST',
+				body: JSON.stringify({ articleId })
+			})
+
+			if (response) {
+				return response
+			}
+			return {}
+		} catch (error) {
+			return { shareToken: undefined }
```

Comment:
**Question:** Why would you return something different here to what you return above on line 70? This comment might become redundant depending on my [question on line 70 ](https://github.com/Financial-Times/x-dash/pull/757/files#r1439310018) but it would be good to keep consistency so raising it anyway

Reply:
yeah, i think just using `return;` should be good

Reply:
@MMSaleh-FT if you change it to `return;` without also changing how its used [here](https://github.com/Financial-Times/x-dash/pull/757/files#diff-7082a9e3cb317538577308da954ff347e6c58e103516faaa15ec7c405cba1161R49) it will result in a destructuring error.